README.md

```diff
@@ -83,57 +83,73 @@ These instructions are tested and verified on macOS Catalina (10.15)
     ```shell
     brew install redis
     ```
+    
+    Run the redis server using command:
+    ```
+    redis-server --daemonize yes
+    ```
+    
 
 #### Development Environment
 
 1. Make sure you have Xcode installed
 
-1. Install a Ruby manager - if you don't already have one, you will need a Ruby manager to install the appropriate Ruby release such as:
+2. Install a Ruby manager - if you don't already have one, you will need a Ruby manager to install the appropriate Ruby release such as:
```

Comment:
Markdown lint actually wants these to all be 1.

Markdown does the right thing when it renders and just makes them a numbered list and it prevents having to update the list later if you add something to the middle.